src/test/java/com/example/repairs/controllers/HelloWorldControllerTest.java

```diff
@@ -0,0 +1,40 @@
+package com.example.repairs.controllers;
+
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest
+class HelloWorldControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+
+    /**
+     * Tests the return content of the url '/hello' when no name parameter is passed
+     * @throws Exception
+     */
+    @Test
+    void getGreetingWithoutName() throws Exception {
+        mockMvc.perform(get("/hello")).andExpect(status().isOk());
+        mockMvc.perform(get("/hello")).andExpect(content().string("Hello World")).andReturn();
```

Comment:
Can you do multiple `andExpects` on the same line? Right now you're running the code multiple times so you should either:
1) Make 1 test for status code and 1 test for the content
2) Make a single test that tests both 
i.e. mockMvc.perform(get("/hello")).andExpect(status().isOk()).andExpect(content().string("Hello World")).andReturn();